### PR TITLE
Add references to face recognition quality attribute

### PR DESCRIPTION
**Pull request by msyache**
## Purpose

* Adds code examples for using the QualityForRecognition attribute recently added to Face Detection API.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: dotnet/Face/FaceQuickstart.cs [hunk 1]
```diff
@@ -402,6 +406,25 @@ public static async Task IdentifyInPersonGroup(IFaceClient client, string url, s
...
+                        if (faceQualityForRecognition.HasValue && (faceQualityForRecognition.Value != QualityForRecognition.High)){
+                            sufficientQuality = false;
+                            break;
+                        }
+                    }
+
+                    if (!sufficientQuality){
+                        continue;
+                    }
+
```
Comment (lucia-msft) [inline dotnet/Face/FaceQuickstart.cs:427]:
```suggestion
                // Add face to the person group person.
                foreach (var similarImage in personDictionary[groupedFace])
                {
                    Console.WriteLine($"Check whether image is of sufficient quality for recognition");
                    IList<DetectedFace> detectedFaces = await client.Face.DetectWithUrlAsync($"{url}{similarImage}", 
                        recognitionModel: recognition_model, 
                        detectionModel: DetectionModel.Detection03,
                        returnFaceAttributes: new List<FaceAttributeType> { FaceAttributeType.QualityForRecognition });
                    bool sufficientQuality = true;
                    foreach (var face in detectedFaces)
                    {
                        var faceQualityForRecognition = face.FaceAttributes.QualityForRecognition;
                        //  Only "high" quality images are recommended for person enrollment
                        if (faceQualityForRecognition.HasValue && (faceQualityForRecognition.Value != QualityForRecognition.High)){
                            sufficientQuality = false;
                            break;
                        }
                    }

                    if (!sufficientQuality){
                        continue;
                    }


```

Comment (lucia-msft) [reply dotnet/Face/FaceQuickstart.cs:427]: Added a comment that only high quality images are recommended for person enrollment

File: javascript/Face/sdk_quickstart.js [hunk 2]
```diff
@@ -42,7 +42,7 @@ async function DetectFaceExtract() {
 	await Promise.all (image_file_names.map (async function (image_file_name) {
```
Comment (lucia-msft) [inline javascript/Face/sdk_quickstart.js:42]: It seems not all languages in the QS have quality filter logic for enrollment and recognition?

Comment (msyache) [reply javascript/Face/sdk_quickstart.js:42]: Correct. I'm not comfortable committing sample code in both languages that I am unfamiliar with and that are currently untestable due to the client updates still being in progress. We could revisit once the client update happens for them.

**Pull request merged**